lib/nostrum/struct/message.ex

```diff
@@ -90,9 +114,32 @@ defmodule Nostrum.Struct.Message do
   @type webhook_id :: Snowflake.t() | nil
 
   @typedoc """
-  [Type of message](https://discord.com/developers/docs/resources/channel#message-object-message-types).
+  |TYPE                                         |	VALUE|
+  |---------------------------------------------|-------|
+  |DEFAULT                                      |	0|
+  |RECIPIENT_ADD                                |	1|
+  |RECIPIENT_REMOVE                             |	2|
+  |CALL                                         |	3|
+  |CHANNEL_NAME_CHANGE                          |	4|
+  |CHANNEL_ICON_CHANGE                          |	5|
+  |CHANNEL_PINNED_MESSAGE                       |	6|
+  |GUILD_MEMBER_JOIN                            |	7|
+  |USER_PREMIUM_GUILD_SUBSCRIPTION              |	8|
+  |USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_1       |	9|
+  |USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_2       |	10|
+  |USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_3       |	11|
+  |CHANNEL_FOLLOW_ADD                           |	12|
+  |GUILD_DISCOVERY_DISQUALIFIED                 |	14|
+  |GUILD_DISCOVERY_REQUALIFIED                  |	15|
+  |GUILD_DISCOVERY_GRACE_PERIOD_INITIAL_WARNING |	16|
+  |GUILD_DISCOVERY_GRACE_PERIOD_FINAL_WARNING   |	17|
+  |THREAD_CREATED                               |	18|
+  |REPLY                                        |	19|
+  |APPLICATION_COMMAND                          |	20|
+  |THREAD_STARTER_MESSAGE                       |	21|
+  |GUILD_INVITE_REMINDER                        |	22|
   """
```

Comment:
This shows up fine compared to the channel type table from above. Maybe some markdown syntax problem?

That said, do you think it would be useful to keep a link to the Discord documentation in here for further reference?

Reply:
I had inconsistent results with rendering markdown tables inside the typedoc section vs normal section which seems to work all the time. 

I have tried a different method for rendering the tables and have included links where appropriate.